rpc/src/optimistically_confirmed_bank_tracker.rs

```diff
@@ -457,6 +469,7 @@ mod tests {
             &mut highest_confirmed_slot,
             &mut newest_root_slot,
             &None,
+            &Arc::new(PrioritizationFeeCache::default()),
```

Comment:
nit: on these tests we no longer need `Arc` since the type passes is just `&PrioritizationFeeCache`